code/style.css

```diff
@@ -1,7 +1,254 @@
+  width: 100vw;
+  height: 100vh;
+  object-fit: cover;
+  position: absolute;  
+  z-index: 1;
+  filter: opacity(60%);
+}
+*/
+.welcome {
+  z-index: 2;
+  position: relative;
+  color: white;
+  font-family: fantasy;
+  z-index: 2;
+  text-align: center;
+  justify-content: center;
+  padding-top: 50%;
+  margin: 0px;
+}
+
+.welcome-title {
+  font-family: "Caveat", cursive;
+  font-weight: 500;
+  font-size: 60px;
+  cursor: crosshair;
+  margin: 0px;
+}
+
+.welcome-title:hover {
+  opacity: 0.5;
+}
+
+.welcome-sub {
+  font-family: "Dosis", sans-serif;
+  cursor: crosshair;
+}
+
+.welcome-sub:hover {
+  opacity: 0.5;
+} 
+
+/* MAIN SECTION */
 
-.active + .answer {
+.main {
+  color: rgb(186, 186, 228);
+  font-family: "Caveat", cursive;
+  padding: 20px;
+  text-align: center;
+  width: 300px;
+  margin: auto;
+}
+
+/* SIGN UP SECTION */
+
+.signup {
+  border: #999;
+  border-width: 8px;
+  border-style: double;
+  padding-bottom: 40px;
+  border-radius: 8px;
+  margin-bottom: 15px;
+  padding-right: 10px;
+  padding-top: 20px;
+  background-color: mistyrose;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+}
+
+.birth-month {
+background-color:  rgb(186, 186, 228);
+margin-bottom: 10px;
+border-radius: 8px;
+padding: 4px;
+margin-left: 5px;
+}
+
+.color-header { 
+  display: flex;
+  justify-content: center;
+  margin-bottom: 10px;
+  padding: 4px;
+}
+
+.color {
+  background-color:  rgb(186, 186, 228);
+  border-radius: 8px;
+  width: 100px;
+  margin-left: 10px;
+}
+
+.text {
+  background-color:  rgb(186, 186, 228);
+  border-radius: 8px;
+  margin-bottom: 20px;
+  padding: 4px;
+  margin-left: 5px;
+}
+
+.submit {
+  color: white;
+  background-color: rgb(186, 186, 228);
+  border: none;
+  padding: 8px 60px;
+  border-radius: 4px;
+  display: inline-flex;
+  margin-bottom: 10px;
+  font-family: fantasy;
+}
+
+/* ACCORDION SECTION */
+
+.accordion {
+  border: #999;
+  border-width: 8px;
+  border-style: double;
+  border-radius: 8px;
+  background-color: mistyrose;
+  padding-top: 20px;
+}
+
+.accordionHeader {
+cursor: pointer;
+padding: 0px;
+}
+
+.accordionHeaderTwo {
+  cursor: pointer;
+  padding: 0px;
+}
+
+.accordionHeaderThree {
+  cursor: pointer;
+  padding: 0px;
+}
+
+.answer {
+  display: none;
+}
+
+.answerTwo {
+  display: none;
+}
+
+.answerThree {
+  display: none;
+}
+
+.active {
+  display: contents;
+}
+
+.activeTwo {
+  display: contents;
+}
+
+.activeThree {
+  display: contents;
+}
+
+
+/* FOOTER */
+
+.footer {
+  background-color: lightgray;
+  color: antiquewhite;
+  text-align: center;
+  padding: 15px;
+  font-family: fantasy;
+  margin-top: 26px;
+}
+
+/* .active + .answer {
   /* some styling here */
+  /* display: block; */
+
+
+/* MEDIA QUERIES TABLET */
+
+@media (min-width: 768px) {
+  
+  .welcome {
+  padding-top: 20%;
+  padding-left: 60%;
+  }
+
+  .welcome-title {
+    font-size: 150px;
+    margin: 0px;
+  }
+
+  .welcome-sub {
+    font-size: 50px;
+    margin: 0px;
+  }
```

Comment:
A little too wide content in tablet:
<img width="638" alt="Skärmavbild 2021-02-09 kl  18 03 37" src="https://user-images.githubusercontent.com/51529201/107399560-241dbc80-6b01-11eb-9c61-6f5e9daae939.png">